pymc3/sampling.py

```diff
@@ -772,8 +800,6 @@ def init_nuts(init='auto', njobs=1, n_init=500000, model=None,
         var = np.ones_like(mean)
         potential = quadpotential.QuadPotentialDiagAdapt(
             model.ndim, mean, var, 10)
-        if njobs == 1:
```

Comment:
I dont think you can delete this.

Reply:
I made changes for this in the appropriate functions (I think, let's see what the tests say :-) )

Reply:
OK I saw it above now :+1: